src/main/benchmark_init.c

```diff
@@ -98,12 +114,17 @@ static struct option long_options[] = {
 	{"compress",              no_argument,       0, '4'},
 	{"compression-ratio",     required_argument, 0, '5'},
 	{"socket-timeout",        required_argument, 0, '1'},
+	{"connect-timeout",       required_argument, 0, CONNECT_TIMEOUT}, // flag matches Java Benchmark, but controls conn_timeout_ms
 	{"read-socket-timeout",   required_argument, 0, '2'},
 	{"write-socket-timeout",  required_argument, 0, '3'},
 	{"timeout",               required_argument, 0, 'T'},
 	{"read-timeout",          required_argument, 0, 'X'},
 	{"write-timeout",         required_argument, 0, 'V'},
 	{"max-retries",           required_argument, 0, 'r'},
+	{"max-error-rate",        required_argument, 0, MAX_ERROR_RATE},
+	{"tender-interval",       required_argument, 0, TENDER_INTERVAL},
+	{"error_rate_window",     required_argument, 0, ERROR_RATE_WINDOW},
```

Comment:
This should be dash-delimited

Reply:
[6dec2be](https://github.com/aerospike/aerospike-benchmark/pull/67/commits/6dec2be036d5908953480991fc04f5f8ed6b7bf0)